src/PioneerUtils.jsm

```diff
@@ -7,7 +7,8 @@ Cu.import("resource://gre/modules/XPCOMUtils.jsm");
 const { TelemetryController } = Cu.import("resource://gre/modules/TelemetryController.jsm", null);
 const { generateUUID } = Cc["@mozilla.org/uuid-generator;1"].getService(Ci.nsIUUIDGenerator);
 
-import { setCrypto, Jose, JoseJWE } from "jose-jwe-jws/dist/jose-commonjs.js";
+import { setCrypto as Jose_SetCrypto, Jose, JoseJWE } from "jose-jwe-jws/dist/jose-commonjs.js";
```

Comment:
you've got `Jose_SetCrypto` here but `Jose_setCrypto` below. Maybe just got with `joseSetCrypto`?

Reply:
Oops, nice catch. It would probably be good to set up some linting here eventually.